data/gen_data.C

```diff
@@ -4,22 +4,22 @@
 void gen_data(){
 	//Create file to store trees
 	TFile* Ofile = TFile::Open("data.root","RECREATE");
-	
+
```

Comment:
The reason why I chose to use a ROOT file here is because I based my work on a c++/ROOT version of BumpHunter.
I needed some data in ROOT format for the test and cross-checks against the model version.
But now that the checks are done, maybe it is not needed anymore.

Reply:
Yes sure, that makes sense. I also think that many things including names come from an earlier version and from trying out. Just pointing out the things that now, making it a mature package, could be changed.
It could also stay a root file oc, but preferably produced with numpy (as you see in the tests it's a fraction of the C++ code) and dumped with uproot or similar

Reply:
BTW, do not forget about our [`scikit-hep-testdata`](https://github.com/scikit-hep/scikit-hep-testdata) package to keep testing files across the org (packages). You could maybe move a ROOT test file there, even with instructions for how to produce it, see the last PR in the package.